target/linux/ramips/base-files/etc/diag.sh

```diff
@@ -157,6 +157,9 @@ get_status_led() {
 	zorlik,zl5900v2)
 		status_led="$boardname:blue:power"
 		;;
+	mikrotik,rbm33g)
```

Comment:
alphabetical ordering please

Reply:
Whoops, didn't quite catch the ordering there, thanks